README.md

```diff
@@ -1,21 +1,100 @@
+| Users & Roles & multi-team support                                             |                                                          | x                                                                             |
+| Authentication & Authorization                                                 | Hosting Webserver                                        | Hosting Webserver<br />LDAP<br />GitHub OAUTH & SAML<br />GitLab OAUTH & SAML |
+| Advanced Data / Filters for Reports and Viewing                                |                                                          | x                                                                             |
+| Multiple Report Output<br />(DISA Checklist XML, CAT, XCCDF-Results, and more) |                                                          | x                                                                             |
+| Authenticated REST API                                                         |                                                          | x                                                                             |
+| InSpec Run 'Delta' View                                                        |                                                          | x                                                                             |
+| Multi-Report Tagging, Filtering and Delta View                                 |                                                          | x                                                                             |
+
+### Use Cases
+
+| [Heimdall-Lite](https://github.com/mitre/heimdall-lite/) | [Heimdall](https://github.com/mitre/heimdall/)           |
+| :------------------------------------------------------- | :------------------------------------------------------- |
+| Ship the App & Data via simple Email                     | Multiple Teams Support                                   |
+| Minimal Footprint & Deployment Time                      | Timeline and Report History                              |
+| Local or disconnected Use                                | Centralized Deployment Model                             |
+| One-Time Quick Reviews                                   | Need to view the delta between one or more runs          |
+| Decentralized Deployment                                 | Need to view subsets of the 800-53 control alignment     |
+| Minimal A&A Time                                         | Need to produce more complex reports in multiple formats |
+
+## General Deployment
+
+### NodeJS Deployment
+
+Heimdall Lite is a standard VueJS app so for help with a local deployment, please see: <https://cli.vuejs.org/guide/deployment.html#general-guidelines>
+
+Heimdall Lite is published to the <npmjs.org> MITRE site at <https://www.npmjs.com/package/heimdall-lite>.
+
+#### Running via npm
+
+To run Heimdall Lite locally, just use the `npm` built-in utility `npx`:
+
+`npx heimdall-lite`
```

Comment:
Should this be `npm run lite:dev` as described in the current `package.json`?

Reply:
This is copy-pasted from the old README.md, my reason for keeping it is it is still relevant as soon as we add the Github Actions to build https://www.npmjs.com/package/heimdall-lite